lib/elixir/lib/calendar/iso.ex

```diff
@@ -10,7 +10,11 @@ defmodule Calendar.ISO do
   """
 
   @behaviour Calendar
+
   @unix_epoch :calendar.datetime_to_gregorian_seconds {{1970, 1, 1}, {0, 0, 0}}
+  @unix_start 1_000_000 * -@unix_epoch
+  @unix_end 1_000_000 * (-@unix_epoch + :calendar.datetime_to_gregorian_seconds({{9999, 12, 31}, {23, 59, 59}}))
+  @unix_range_microsecs @unix_start..@unix_end
```

Comment:
Doesn't seem we save much with using `microsecs` instead of fully-spelled `microseconds`.